src/OrchardCore.Modules/OrchardCore.Recipes/Controllers/AdminController.cs

```diff
@@ -105,19 +105,7 @@ public async Task<ActionResult> Execute(string basePath, string fileName)
 
             var executionId = Guid.NewGuid().ToString("n");
 
-            // Set shell state to "Initializing" so that subsequent HTTP requests
```

Comment:
@Skrypt when #12375 is merged, you can restore back those lines

Reply:
Mouse and cat game. I'm not sure if that #12375 fix is the proper one. Maybe for backward compatibility but else ...